### PR TITLE
rating and snackbar

### PR DESCRIPTION
**Pull request by laxmikanthkonthum**
**Snackbar:**
image
**Rating:**
image

### CODE REVIEW

File: rating - snackbar/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import url(/style.css);
+
+.snack {
+    position: fixed;
+    color: var(--secondary-bg);
+    background-color: #44403c;
+    padding: 0.8rem;
```
Comment (anupkgurung) [inline rating - snackbar/snackbar/snackbar.css:7]: you can move hex code to where --secondary-bg is defined.

**Review by anupkgurung — APPROVED** [review]
Looks good. Please look at the comment.

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Comment by ckahub** [comment]
looks good to me

**Review by ckahub — COMMENTED** [review]
looks good to me

**Comment by ckahub** [comment]
you can go ahead and merge

File: rating - snackbar/snackbar/snackbar.css (hunk 1 above)
Comment (laxmikanthkonthum) [reply rating - snackbar/snackbar/snackbar.css:7]: yeah will do it, since it is a one time usage i did not declare it that way

**Pull request merged**